cmd/neofs-adm/internal/modules/morph/generate.go

```diff
@@ -42,6 +42,11 @@ func generateAlphabetCreds(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	err = initializeManifestWallet(walletDir)
+	if err != nil {
+		return err
+	}
```

Comment:
Correct me if I am wrong: in the case when we already have all alphabet wallets, we call `generate-alphabet` to generate new wallets but we will take only new manifest wallet from that.

Reply:
I think we can initialize it during deploy if it is missing.

Reply:
So `generate-alphabet` will not rewrite existing wallets now, right? Or it needs to be implemented separately?

Reply:
Current behaviour:
1. `generate-alphabet` creates alphabet wallets and a contract wallet, overwrites all existing files.
2. `morph init` opens all wallets from disk and creates a contract wallet if it is missing.